Documentation/operations/upgrade.rst

```diff
@@ -178,13 +174,13 @@ the initial deployment:
 
       helm upgrade cilium |CHART_RELEASE| \\
         --namespace=kube-system \\
-        --set config.upgradeCompatibility=1.7 \\
+        --set upgradeCompatibility=1.7 \\
         --set agent.keepDeprecatedProbes=true
 
 .. note::
 
    Make sure that you are using the same options as for the initial deployment.
-   Instead of using ``--set``, you can also modify the ``values.yaml`` in
+   Instead of using ``--set ``, you can also modify the ``values.yaml`` in
```

Comment:
there's a white space after `--set` that shouldn't be there.

Reply:
fixed